evm/its.js

```diff
@@ -57,6 +58,24 @@ async function handleTx(tx, chain, contract, action, firstEvent, secondEvent) {
     }
 }
 
+const decodeMulticallData = async (encodedData, contractJSON) => {
+    const decodedArray = defaultAbiCoder.decode(['bytes[]'], encodedData)[0];
+    const iface = new Interface(contractJSON.abi);
+
+    return decodedArray.map((encodedCall) => {
+        try {
+            const parsedCall = iface.parseTransaction({ data: encodedCall });
+            const functionName = parsedCall.name;
+            const args = parsedCall.args.map((arg) => arg.toString());
```

Comment:
can you get the name of the args as well? So we can print
Function name
Arg name list
Arg value list

Reply:
added